peewee_extra_fields/__init__.py

```diff
@@ -1153,6 +1153,32 @@ def python_value(self, value: str) -> namedtuple:
         return value
 
 
+class EnumField(CharField):
+    """
+    This class enables a Enum like field for Peewee.
+    """
+    db_field = "enum"
```

Comment:
Is this needed? I took it just from the code examples.

Reply:
`db_field` determines which kind of field to use on the Database, not Python nor Peewee but a raw sql database field.

I hope `Enum` should be Ok since its documented https://www.postgresql.org/docs/current/static/datatype-enum.html

Reply:
AFAIK enum is not supported in sqlite3.

Reply:
Main support focus is PosgreSQL, as long as it works on PostgreSQL it should be Ok, 
because most developers use PostgreSQL, and SQLite for quick tests and hello worlds.